src/model/error.messages.ts

```diff
@@ -10,8 +10,10 @@ export const ORDERS_DETAILS_FIRST_NAME_EMPTY: string = "Enter your first name";
 export const ORDERS_DETAILS_LAST_NAME_EMPTY: string = "Enter your last name";
 export const ORDER_DETAILS_FIRST_NAME_MAX_LENGTH: string = "First name must be 32 characters or fewer";
 export const ORDER_DETAILS_LAST_NAME_MAX_LENGTH: string = "Last name must be 32 characters or fewer";
+export const ORDER_DETAILS_COMPANY_NAME_NAME_MAX_LENGTH: string = "Company name must be 32 characters or fewer";
```

Comment:
Are we sure about this error? Company Names can be up to 160 Characters long in the db

Reply:
I asked Frank about this. After consultations with Barri (and Rob?), he asked me to make it 32 like the other cases for now. Stories will be created in the future to make it longer throughout.